main.go

```diff
@@ -0,0 +1,22 @@
+//A very simple Go application.
```

Comment:
nit: add a space after the '//' (here and in comments below too)

Reply:
Thanks... I feel like some code editor insisted I remove them at some point!